packages/vuetify/src/composables/display.ts

```diff
@@ -170,9 +175,9 @@ export function createDisplay (options?: DisplayOptions, ssr?: boolean): Display
       : xl ? 'xl'
       : 'xxl'
     const breakpointValue = typeof mobileBreakpoint === 'number' ? mobileBreakpoint : thresholds[mobileBreakpoint]
-    const mobile = !platform.ssr
+    const mobile = !platform.value.ssr
       ? width.value < breakpointValue
-      : platform.android || platform.ios || platform.opera
+      : platform.value.android || platform.value.ios || platform.value.opera
```

Comment:
Unrelated but this seems kinda off. If ssr the other three will always be false, so really it's just checking width < mobileBreakpoint except that would be true for ssr